src/features/pages/home/live-pricing/table-component/live-market-table.tsx

```diff
@@ -13,37 +13,56 @@ import Flex from 'features/components/atoms/flex-box'
 import useRegion from 'components/hooks/use-region'
 import Link from 'features/components/atoms/link'
 import { Localize } from 'components/localization'
+import InitialLoader from 'components/elements/dot-loader'
+import useNetwork from 'features/hooks/use-network'
 
 export type TLiveMarketTableProps = {
     selected_market: TAvailableLiveMarkets
-    link_to: string
+    link_to: `/${string}`
+}
+
+type MarketResponseType = {
+    market: {
+        [key in TAvailableLiveMarkets]: {
+            [key: string]: TMarketData
+        }
+    }
 }
 
 const LiveMarketTable = ({ selected_market, link_to }: TLiveMarketTableProps) => {
     const { is_eu } = useRegion()
     const [initial_loaded, setInitialLoaded] = useState(false)
-
-    const [rawMarketsData, setRawMarketsData] = useState()
-    const TABLE_VISIBLE_ROWS = 5
+    const [rawMarketsData, setRawMarketsData] = useState<MarketResponseType>()
+    const [sorting, setSorting] = React.useState<SortingState>([])
+    const intervalRef = useRef(null)
     const [markets_data, setMarketsData] = useState(() => {
         const temp = new Map<TAvailableLiveMarkets, TMarketData[]>()
         return temp
     })
-    const intervalRef = useRef(null)
 
-    const [sorting, setSorting] = React.useState<SortingState>([])
+    const network = useNetwork()
```

Comment:
since if needed to check when user had connection last time
```suggestion
    const {online, effectiveType, since} = useNetwork()
```